index.html

```diff
@@ -1,12 +1,161 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title></title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Alumni+Sans+Pinstripe&family=Crete+Round:ital@0;1&family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto+Flex:opsz,wght@8..144,100;8..144,200;8..144,300;8..144,400;8..144,600;8..144,700;8..144,800;8..144,900;8..144,1000&display=swap"
+    rel="stylesheet">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <link rel="stylesheet" href="main.css">
+  <title>Capstone</title>
 </head>
+
 <body>
-  
+  <header class="container=fluid">
+    <nav class="navbar nav1 navbar-expand-lg fixed-top nav_container d-inline-flex justify-content-end pb-0 pt-0"
+      role="navigation">
+      <ul class="">
+        <li><a href="#" class="nav-link1"><img src="assets/images/104498_facebook_icon.svg" alt="facebook icon"
+              class="nav-icons"></a></li>
+        <li><a href="#" class="nav-link1"><img src="assets/images/104501_twitter_icon.svg" alt="twitter icon"
+              class="nav-icons"></a></li>
+        <li><a href="#" class="nav-link1">English</a></li>
+        <li><a href="#" class="nav-link1">My Page</a></li>
+        <li><a href="#" class="nav-link1">Logouts</a></li>
+      </ul>
+    </nav>
+    <a class="menu justify-content-start"><img src="assets/images/menu_button.svg" alt="menu button"></a>
+
+    <nav class="navbar nav2 navbar-expand-lg fixed-top nav_container d-inline-flex justify-content-end"
+      role="navigation">
+      <ul class="">
+        <li id="small-logo"><a href="index.html"><img class="logo-small" src="assets/images/BTR-Logo.png" alt="btr logo"></a></li>
```

Comment:
- [ ] As the last code reviewer said: the logo is too far to the right. Please 🙏🏽 align it to look like Cindy´s design.
![image](https://user-images.githubusercontent.com/58642949/192122930-3f3d6486-688e-413a-b2f5-d9f145412576.png)